ansible/roles/ocp-workload-migration/templates/controller-3.yml.j2

```diff
@@ -0,0 +1,15 @@
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
```

Comment:
@djwhatle best to use mig_migration_namespace var here

Reply:
Fixed